problems/4.arrays.js

```diff
@@ -3,25 +3,31 @@
 * @returns {number} - The average of the first and last values
 */
 function firstAndLastAverage() {
-
+// const firstAndLastAverage =  => {
```

Comment:
First element is arr[0] , last is arr[arr.length - 1] Average would be those numbers added together and divided by 2.